humans/richard.txt

```diff
@@ -0,0 +1 @@
+Hi, I'm Richard Abrokwah from Ghana.
```

Comment:
Apologies that I missed this in the first pass. This file would also need to be named as your username: `grandfunkygamer.txt`.